### DISCUSSION TITLE
Is there any options to override status bar to show only ideavim plugin status?

### DISCUSSION THREAD

**Discussion by momadacoding** [JetBrains/ideavim#373] [Q&A]
![image](https://user-images.githubusercontent.com/2383231/136729391-2149d860-3053-4025-a494-f784a908ee8b.png)

When I start recod micro,
I Just can't know weather if I was in the right vim state, If there was no error, it should show "VIM - recoding"

**Comment by AlexPl292** [comment]
Hi, unfortunately, there is no option for that. I've filed an issue: https://youtrack.jetbrains.com/issue/VIM-2427
